bigquery/unit_tests/test__helpers.py

```diff
@@ -186,7 +186,7 @@ def test_w_scalar_subfield(self):
     def test_w_repeated_subfield(self):
         subfield = _Field('REPEATED', 'color', 'STRING')
         field = _Field('REQUIRED', fields=[subfield])
-        value = {'f': [{'v': ['red', 'yellow', 'blue']}]}
+        value = {'f': [{'v': [{'v': 'red'}, {'v': 'yellow'}, {'v': 'blue'}]}]}
```

Comment:
It was an incorrect guess at how the nested array was represented (the docs have no examples of the mapping).